sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Listeners/QueueTargetScaler.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Storage.Queues;
+using Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners
+{
+    /// <summary>
+    /// Provides queue length metrics for the <see cref="QueueTargetScaler"/>.
+    /// </summary>
+    internal sealed class QueueTargetScaler : ITargetScaler
```

Comment:
Please add a test for the class, something like:
https://github.com/Azure/azure-sdk-for-net/pull/31367/files#diff-fa639f441f0f5a55d5a76231d750b00594a4197a9f20fa58bc18825cd8109de2

Reply:
added tests for TargetScaler